lib/archethic/utils/regression/benchmarks/helpers/tps_helper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ArchEthic.Utils.Regression.Benchmarks.Helpers.TPSHelper do
+  @moduledoc """
+  Helpers Methods to carry out the TPS benchmarks
+  """
+
+  def withdraw_uco_from_testnet_endpoint(recipient_address) do
+    alias ArchEthic.Utils.Regression.Benchmarks.Helpers.FaucetEndpoint
+    FaucetEndpoint.main(recipient_address)
+  end
+
+  def withdraw_uco_via_this_node(recipient_address) do
+    alias ArchEthic.Utils.Regression.Benchmarks.Helpers.InternalFaucet
+    InternalFaucet.main(recipient_address)
+  end
+
+  def withdraw_uco_via_host(recipient_address, host, port) do
```

Comment:
This is sufficient as @roychowdhuryrohit-dev mentioned. You should target any node. 
Because the benchmark can be run with a single node or with a docker compose network.